src/mare/compiler/macros.cr

```diff
@@ -139,6 +139,12 @@ class Mare::Compiler::Macros < Mare::AST::CopyOnMutateVisitor
         "the value whose identity is to be hashed",
       ])
       visit_identity_digest_of(node)
+    elsif Util.match_ident?(node, 0, "address_of")
```

Comment:
Can you add a `spec/compiler/macros/address_of_spec.cr` file that has tests for this?

You should be able to copy tests from `spec/compiler/macros/identity_digest_of_spec.cr` and just adjust the text.

Reply:
Sure